mteb/tasks/Classification/__init__.py

```diff
@@ -24,6 +24,13 @@
 from .eng.ContractNLIExplicitIdentificationLegalBenchClassification import *
 from .eng.ContractNLIInclusionOfVerballyConveyedInformationLegalBenchClassification import *
 from .eng.ContractNLILimitedUseLegalBenchClassification import *
+from .eng.ContractNLISharingWithThirdPartiesLegalBenchClassification import *
```

Comment:
Can you please add all tasks in one file like [here](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/Classification/multilingual/ScalaClassification.py)

Reply:
Sure, I'll do that!